crt_portal/cts_forms/forms.py

```diff
@@ -901,6 +900,9 @@ class Filters(ModelForm):
         })
     )
 
+    def __init__(self, *args, **kwargs):
+        ModelForm.__init__(self, *args, **kwargs)
```

Comment:
:thinking: Any particular reason we need this? I removed and seems to work fine ...